app/buckets/handlers.js

```diff
@@ -42,7 +42,8 @@ exports.bucket_details = (req, res, next) => {
         bucket,
         apps_options: apps.exclude(bucket.apps),
         users_options: users.exclude(bucket.users),
-        current_user_is_bucket_admin
+        current_user_id: req.user.auth0_id,
+        current_user_is_bucket_admin,
```

Comment:
You could replace `current_user_is_bucket_admin` in the template with `bucket.has_admin(current_user)`, rather than assigning a variable and passing it.

Or if you add `is_admin(bucket)` to the `User` model, you could  `current_user.is_admin(bucket)`.

Reply:
I completely forgot about this `Bucket#has_admin` method 😇

Reply:
👍